packages/shared/chat/sagas.ts

```diff
@@ -75,9 +75,9 @@ function* trackEvents(action: PayloadAction<MessageEvent, ChatMessage>) {
       trackEvent('Send chat message', {
         messageId: payload.messageId,
         from: hasWallet() ? ChatMessagePlayerType.WALLET : ChatMessagePlayerType.GUEST,
-        to: ChatMessagePlayerType.WALLET,
+        to: payload.messageType === ChatMessageType.PRIVATE ? undefined : ChatMessagePlayerType.WALLET,
```

Comment:
This logic is not clear, what does "to" stands for? From my understanding if the message type is PRIVATE, then `to` should be WALLET, what is the second option? what does undefined stands for?

Reply:
"to" is the recipient of the message. I agree `recipient` would be more describing but since we already have events tracked with that field as "to" I would suggest to keep it this way.

You are right! I misplaced the ternary operator, it should be WALLET for PRIVATE msgs and undefined (because there's no recipient) for the rest.